setup.py

```diff
@@ -55,6 +55,10 @@ def read_file(filename, encoding='utf8'):
         'argparse',
         'mock<1.1.0',
     ])
+elif sys.version_info < (2, 8):
```

Comment:
Shouldn't this be `sys.version_info < (2, 7, 9)`?

https://urllib3.readthedocs.org/en/latest/security.html#installing-urllib3-with-sni-support-and-certificates